test/acceptance/fs.js

```diff
@@ -1,16 +1,18 @@
 var fs = require('fs');
+var path = require('path');
+var t = path.join.bind(path, require('os-tmpdir')());
 
 describe('fs.readFile()', function(){
   describe('when the file exists', function(){
     it('should succeed', function(done){
-      fs.writeFile('/tmp/mocha', 'wahoo', done)
+      fs.writeFile(t('mocha'), 'wahoo', done)
     })
   })
 
   describe('when the file does not exist', function(){
     it('should fail', function(done){
       // uncomment
-      // fs.readFile('/tmp/does-not-exist', done);
+      // fs.readFile(t('does-not-exist'), done);
       done();
     })
   })
```

Comment:
Note to self: go through the test suites and find things like this that are failure-tests but either commented out or otherwise unusually handled and if possible change them to assert that the failure throws or passes an error to a callback or whatever the expectable failure mode is.